### DISCUSSION TITLE
Reporting: make columns invisible for users

### DISCUSSION THREAD

**Discussion by idemus** [kimai/kimai#3684] [Questions & Answers]
Hi,
is there a possibility to make columns not visible for normal users (role user)?
They should not see columns including rates, earnings, revenues.

![image](https://user-images.githubusercontent.com/71780381/206678107-5e1fc1db-67a9-49dc-9a41-64b843a3b817.png)

Best regards
Klaus

**Comment by kevinpapst** [comment]
They are shown according to your permission setup. 
Are you running the latest version?

**Comment by idemus** [comment]
![image](https://user-images.githubusercontent.com/71780381/206687224-86b190c9-785c-4392-aa02-4a7c0058b7c5.png)

**Reply by kevinpapst** [reply]
You shall not open reports and waste maintainer time before running the latest version

<img width="986" alt="Bildschirmfoto 2022-12-09 um 12 35 01" src="https://user-images.githubusercontent.com/533162/206693916-d4e59c8c-a4d4-406a-9e24-84e2443f739b.png">

**Reply by idemus** [reply]
1.29.1 worked. Thx. 👍 
I don't want to waste anybody's time including my own one but I've searched over one hour on google, on kimai.org and found nothing.

**Reply by kevinpapst** [reply]
Golden rule: always update first 😁 

Edit: I wasted hours myself, reporting stuff that was already fixed. So I know both sides of that "problem"
